packages/fxa-auth-server/lib/account-delete.ts

```diff
@@ -55,17 +49,6 @@ type EnqueueByEmailParam = {
   reason: ReasonForDeletion;
 };
 
-/*
-const isValidDeleteTask = (deleteTask: DeleteTask) => {
-  const uidValidationResult = uidValidator.validate(deleteTask.uid);
-  const customerIdValidationResult = customerIdValidator.validate(
-    deleteTask.customerId
-  );
-
-  return !uidValidationResult.error && !customerIdValidationResult.error;
-};
-//*/
```

Comment:
The payload is validated in the usual hapi way with the validation rules declared for the route.  We don't need this.